### PR TITLE
Updated the individual careers page design

### PR DESCRIPTION
**Pull request by Nikhil-Ladha**
Signed-off-by: Nikhil-Ladha <nikhilladha1999@gmail.com>

**Description**
The design in the PR has a few changes which I think would be better,  from the original Figma design.

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [X] Yes, I signed my commits.

### CODE REVIEW

**Comment by Tanuj22** [comment]
@Nikhil-Ladha Is this PR complete? There is some text-overflow on a card and image is missing on the participating partners

**Comment by Nikhil-Ladha** [comment]
> @Nikhil-Ladha Is this PR complete? There is some text-overflow on a card and image is missing on the participating partners

Yeah, it's complete.
Partner's image were not supposed to be added 🤔

**Comment by Tanuj22** [comment]
> > @Nikhil-Ladha Is this PR complete? There is some text-overflow on a card and image is missing on the participating partners
> 
> Yeah, it's complete.
> Partner's image were not supposed to be added

Isn't this looking weird? :sweat_smile: 
Screenshot from 2021-01-10 23-08-09

**Comment by Nikhil-Ladha** [comment]
> Isn't this looking weird? 
> Screenshot from 2021-01-10 23-08-09

Actually, I didn't notice that previously. Checked it out after you referred to that section 😆 .....pushing an update ASAP.

**Comment by Nikhil-Ladha** [comment]
Done.

**Comment by Tanuj22** [comment]
The text is overflowing...
Screenshot from 2021-01-11 01-34-33

**Comment by Nikhil-Ladha** [comment]
> The text is overflowing...
> Screenshot from 2021-01-11 01-34-33

At what resolution is this happening?
I couldn't find this, when I checked.

**Comment by leecalcote** [comment]
For me, the overflow is happening on three column rows (not in two column rows).

**Comment by Nikhil-Ladha** [comment]
At what resolution is that happening? Any particular range or throughout?

**Comment by Tanuj22** [comment]
For me it's happening when the screen width is greater than 992px. Will be fixed if the text size is kept 16px  ig

**Comment by Nikhil-Ladha** [comment]
> For me it's happening when the screen width is greater than 992px. Will be fixed if the text size is kept 16px ig

Can you do one thing, inspect and check if the width of all the cards is the same or not.....as per the code it should be the same and it shouldn't' happen at all.

**Comment by Tanuj22** [comment]
> > For me it's happening when the screen width is greater than 992px. Will be fixed if the text size is kept 16px ig
> 
> Can you do one thing, inspect and check if the width of all the cards is the same or not.....as per the code it should be the same and it shouldn't' happen at all.
> 
> Because for me it is looking fine in all the resolutions.

Yeah its same :sweat_smile: .

**Comment by Nikhil-Ladha** [comment]
Interesting, so doing what fixes this? According to you.

**Comment by Tanuj22** [comment]
Yeah. Below 992px we the decreasing the heading and paragraph font size globally so that makes it fit in the card

**Comment by Nikhil-Ladha** [comment]
So, suggesting to keep the same font-size throughout for the cards( both heading and text), right?

**Comment by Tanuj22** [comment]
> So, suggesting to keep the same font-size throughout for the cards( both heading and text), right?

Yes

**Comment by Nikhil-Ladha** [comment]
Check if it works now or not ( after the rebuild is complete 😬 )

**Comment by Tanuj22** [comment]
> Check if it works now or not ( after the rebuild is complete  )

The font sizes didn't decrease :grimacing: . Something to look into at the website call? :laughing:

**Comment by Nikhil-Ladha** [comment]
> > Check if it works now or not ( after the rebuild is complete  )
> 
> The font sizes didn't decrease  . Something to look into at the website call?

Yeah, I haven't decreased the font-size but did other adjustments.
Oh, right. Let's discuss it on the websites call. 👍🏼

**Comment by Nikhil-Ladha** [comment]
@leecalcote @Tanuj22 check now.

**Comment by leecalcote** [comment]
@Nikhil-Ladha the issue in Safari is gone.

**Comment by Nikhil-Ladha** [comment]
> @Nikhil-Ladha the issue in Safari is gone.

Finally 😂.... well then if everything is gine we can merge this PR. It's complete.

**Review by leecalcote — APPROVED** [review]

File: src/sections/Internship-gird/InternshipPage.style.js [hunk 1]
```diff
@@ -52,50 +45,90 @@ const InternshipSectionWrapper = styled.div`
...
         h5{
-            color: black; 
-            font-weight: 600;
-            align-self: center;
-            margin: auto 0.5rem;
-            transition: all 0.2s;
+            display: inline-flex;;
+            font-weight: 400;
+            line-height: 3.5rem;
+            vertical-align: middle;
```
Comment (Nikhil-Ladha) [inline src/sections/Internship-gird/InternshipPage.style.js:105]: Ahhh....I think now it makes the text and the button not properly aligned vertically 😬....I know technically this is correct, but visually `top` was working 😆

**Comment by leecalcote** [comment]
You’re right. I meant to, but I forgot to change this back to `top`.

**Pull request merged**